internal/flink/command_statement_resume.go

```diff
@@ -22,23 +22,23 @@ func (c *command) newStatementResumeCommand() *cobra.Command {
 		RunE:              c.statementResume,
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: `Request to resume statement "my-statement" with original principal id and compute pool.`,
+				Text: `Request to resume the currently stopped statement "my-statement" with original principal id and compute pool.`,
```

Comment:
nit: here and below
```suggestion
				Text: `Request to resume the currently stopped statement "my-statement" using original principal id and compute pool.`,
```

![image](https://github.com/user-attachments/assets/b6c7cedc-6fad-4d98-b9b2-c07762c7c751)